### PR TITLE
Lowest precedence for <| and <?|

### PR DESCRIPTION
**Pull request by fsoikin**
By suggestion from @mcordova47, making precedence of `<|` and `<?|` zero, so that they can be used like `dispatch <| foo $ stuff $ more $ stuff`. 

This works, even though `$` also has zero precedence, because `<|` and `<?|` are left-associative, while `$` is right-associative, so mixing them in the same expression is allowed.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ jobs:
           .spago
           output
           node_modules
-        key: build-atrifacts-v2-${{ hashFiles('package-lock.json', 'spago.dhall', 'packages.dhall') }}
+        key: build-artifacts-v2-${{ hashFiles('package-lock.json', 'spago.dhall', 'packages.dhall') }}
```
Comment (fsoikin) [inline .github/workflows/build.yml:22]: Drive-by, thanks to @gasi

**Review by CharlesTaylor-CV — APPROVED** [review]
